tools/build_pytorch_libs.py

```diff
@@ -77,7 +77,7 @@ def cmake_defines(lst, **kwargs):
 
 base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 torch_lib_dir = base_dir + "/torch/lib"
-install_dir = base_dir + "/torch/lib/tmp_install"
+install_dir = os.getenv("INSTALL_DIR", base_dir + "/torch/lib/tmp_install")
```

Comment:
I don't get this change, why do we need a separate INSTALL_DIR variable? I don't see it set anywhere in the codebase. I dislike adding stuff to the build that doesn't  have a clear purpose. It makes it impossible to keep the build simple and fast.

Reply:
Sorry. It's intended for building libtorch on Windows. I'll revert it back.